plugin/storage/es/esCleaner.py

```diff
@@ -15,7 +15,8 @@ def main():
         print('HOSTNAME ... specifies which Elasticsearch hosts URL to search and delete indices from.')
         print('TIMEOUT ...  number of seconds to wait for master node response.'.format(TIMEOUT))
         print('INDEX_PREFIX ... specifies index prefix.')
-        print('ARCHIVE ... specifies whether to remove archive indices (default false).')
+        print('ARCHIVE ... specifies whether to remove archive indices (only works for rollover) (default false).')
```

Comment:
Is this a change in behaviour, or did it never work for archive (with daily indices)?

Reply:
It is just a comment change. 

It worked and works for archive - but it is only supported for archive managed by rollover. If there is one archive we do not touch it.